.github/workflows/grapl-build.yml

```diff
@@ -118,24 +119,17 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Build Grapl
-        run: |
-          GRAPL_RELEASE_TARGET=debug TAG=latest ./dobi-linux --no-bind-mount build
-
-      # --no-bind-mount makes dobi screw up with volumes for some reason, and it's not really required here anyway.
       - name: Run end-to-end tests
         run: |
-          GRAPL_RELEASE_TARGET=debug \
-          GRAPL_LOG_LEVEL=DEBUG \
+          GRAPL_LOG_LEVEL=debug \
           DUMP_ARTIFACTS=True \
-          TAG=latest \
-          ./dobi-linux run-e2e-tests
-       
+          make test-e2e
+
       # NOTE: This requires >= py37
       - name: 'Collect e2e test artifacts'
         if: ${{ always() }}
         run: |
-          python3 ./etc/ci_scripts/dump_artifacts.py --compose-project "grapl-integration-tests"
+          python3 ./etc/ci_scripts/dump_artifacts.py --compose-project "grapl-integration_tests"
```

Comment:
the partial-hyphen-partial-underscore seems odd but I'll take your word for it. 
Ideally we'd change it to something less odd if possible.

Reply:
Agreed, we can change that.

Reply:
ended up sticking with it - my preferred delimiters `.` and `::` are just ignored by docker